config-generators/mssql-commands.txt

```diff
@@ -43,6 +43,7 @@ add MappedBookmarks --config "dab-config.MsSql.json" --source "mappedbookmarks"
 add FteData --config "dab-config.MsSql.json" --source fte_data --permissions "anonymous:*" --graphql "FteData:FteData"
 add InternData --config "dab-config.MsSql.json" --source intern_data --permissions "anonymous:*" --graphql "InternData:InternData"
 add BooksSold --config "dab-config.MsSql.json" --source "books_sold" --rest true --graphql "books_sold:books_sold" --permissions "anonymous:*"
+add DefaultBuiltInFunction --config "dab-config.MsSql.json" --source "default_with_function_table" --rest true --graphql true --permissions "anonymous:*"
```

Comment:
as Ani mentioned, you'll need a test to validate the erroneous behavior reported in the issue. 
add permissions that exclude a column that has a default value as a built-in function, and ensure that the request executes as expected.